src/docs/patterns/tags.mdx

```diff
@@ -0,0 +1,230 @@
+import MultiTagInfoLine from '../../app/assets/design-patterns/tags/multi-tag-info-line.png';
+import MultiTagEllipsis from '../../app/assets/design-patterns/tags/multi-tag-ellipsis.png';
+import MultiTagWrap from '../../app/assets/design-patterns/tags/multi-tag-wrap.png';
+import MultiTagWrapInline from '../../app/assets/design-patterns/tags/multi-tag-wrap-inline.png';
+import Spec1 from '../../app/assets/design-patterns/tags/spec-1.svg';
+import Spec2 from '../../app/assets/design-patterns/tags/spec-2.svg';
+
+{/* KEYWORDS: design pattern tag status badge label */}
+
+# Tags
+
+<ImageGrid images={[BannerImage]} regularWidth />
+
+Tags, sometimes known as badges and labels, are UI components to indicate statuses with text labels and status colors.
+
+<TOC
+    anchors={[
+        { title: 'Introduction', hash: '#chips' },
+        { title: 'Tags vs. Chips' },
+        { title: 'Common Use Cases' },
+        { title: 'Variants' },
+        { title: 'Colors' },
+        { title: 'Labels' },
+        { title: 'Text-Only', depth: 1 },
+        { title: 'Short Labels', depth: 1 },
+        { title: 'Multiple Tags' },
+        { title: 'Design Specifications' },
+        { title: 'Developers' },
+    ]}
+/>
+
+# Tags vs. Chips
+
+<ImageGrid
+    images={[TagsNChipsTag, TagsNChipsChips]}
+    captionsUnderImages={[`Tags — static and non-interactive.`, `Chips — contextual and interactive.`]}
+/>
+
+The name "tag" may sometimes be used for another UI component "chip", but they serve different purposes. Tags are static and non-interactive. They are commonly used as status indicators or something to capture users’ attention. Chips are versatile, contextual, and interactive. They are used for filtering or suggested actions. Chips may also contain auxiliary elements like leading icons or trailing remove icons.
+
+This guide deals specifically with the design of tags. For chips, please refer to [Material Design's guidelines](https://m2.material.io/components/chips).
+
+<MaterialDesignDescription title={'Chips'} url={`https://m2.material.io/components/chips.html`} />
+
+<Divider />
+
+# Common Use Cases
+
+Tags are nearly exclusively seen in [list items](/patterns/lists) to annotate a status such as alarms, warnings, active, and acknowledged. It can also be used to call for visual attention (e.g., “Update Required”).
+
+<ImageGrid
+    images={[InList, InAlarms, InList2, InTimestamp, Inline, InTables]}
+    caption={`Tags used inside different components.`}
+/>
+
+Some designers may choose to use icons to depict statuses, but tags require less “guesswork” from users. Although there are a few "[universal icons](/style/icon-guidelines#common-universal-icons)" that are instantly recognizable and should stay as icons, If users are not expected to frequently interact with your application, they should not be forced to memorize all the rare icons and their meanings — and tags are wonderful for avoiding this situation.
```

Comment:
```suggestion
Some designers may choose to use icons to depict statuses, but tags are more explicit and don't require users to recall or guess the meaning. There are a few [universal icons](/style/icon-guidelines#common-universal-icons) that are instantly recognizable and should stay as icons, but for concepts that may be less familiar, tags are a wonderful way to avoid any confusion.
```